lib/Frontend/ModuleInterfaceSupport.cpp

```diff
@@ -130,7 +130,7 @@ static void printImports(raw_ostream &out,
 
     // SPI attribute on imports
     if (Opts.PrintSPIs) {
-      SmallVector<Identifier, 4> spis;
+      llvm::SmallSet<Identifier, 4> spis;
```

Comment:
Can we use `llvm::SetVector` instead of `llvm::SmallSet`? So that the printed SPI attributes are always in a deterministic order.

Reply:
Good idea, I updated it to use a `SmallSetVector`.

Reply:
Thank you!